elements/pfe-navigation/src/pfe-navigation--lightdom.scss

```
@@ -23,7 +23,7 @@
         &.expanded {
             @media screen and (min-width: #{pfe-breakpoint(lg)}) {
                 background-color: #{pfe-var(surface--lightest)};
-                border-top-color: #{pfe-var(surface--accent)};
+                border-top-color: #{pfe-var(ui-accent)};
```

```suggestion
                border-top-color: pfe-var(ui-accent);
```
I know you didn't introduce this but if you wanted to, you could remove the interpolation here #pebble

---

updated